docs/CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ The following tip changes can be tested by building VictoriaMetrics components f
 * BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): fix application routing issues and problems with manual URL changes. See [this pull request](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/4408).
 * BUGFIX: add validation for invalid [partial RFC3339 timestamp formats](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#timestamp-formats) in query and export APIs.
 * BUGFIX: [vmctl](https://docs.victoriametrics.com/vmctl.html): interrupt explore procedure in influx mode if vmctl found no numeric fields.
+* BUGFIX: [vmctl](https://docs.victoriametrics.com/vmctl.html): remove undefined flag `--remote-read-src-check-alive=false` from documentation. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4552).
```

Comment:
Seems like it is purely a documentation update, do we need to have a changelog entry for this?

Reply:
@zekker6 , we have an issue describing these changes, so I decided to add a changelog with the issue mentioned. If it is not needed I will remove